src/elements/content-sidebar/DetailsSidebar.scss

```diff
@@ -3,6 +3,10 @@
 .bcs-details {
     .bcs-scroll-content {
         padding-top: 15px;
+
+        > .collapsible-card {
+            margin: 0 8px 20px 25px;
```

Comment:
Can the collapsible panel be wrapped in a SidebarSection to avoid duplicating these styles?

Reply:
Depends, if we refactor that to just use collapsible then yeah. But if we just nuke that and use collapsible directly, then over here will work for all "cards" in the details sidebar.